### ISSUE TITLE
content_issue @ "Introduction to Classification Algorithms" by dellal.khaled.enssea

### ISSUE THREAD

**Issue by datawars-content** [datawars-io-content/content-feedback#6]
I got the following error : ImportError cannot import name 'plot_confusion_matrix' from 'sklearn.metrics'

**Comment by datawars-content** [comment]
❌ Must assign one (and only one) resolution label. Reopening issue...

**Comment by datawars-content** [comment]
❌ Must assign one (and only one) resolution label. Reopening issue...

**Comment by datawars-content** [comment]
❌ Must assign one (and only one) resolution label. Reopening issue...
